core/controllers/dashboard.py

```diff
@@ -204,6 +204,7 @@ def _round_average_ratings(rating):
                     'created_on': utils.get_time_in_millisecs(
                         collection_summary.collection_model_created_on),
                     'status': collection_summary.status,
+                    'node_count': collection_summary.node_count,
```

Comment:
Could it be done in this PR? I don't want to continue proliferating the duplication.